lib/adapters/sql.ex

```diff
@@ -181,7 +181,7 @@ defmodule Apartmentex.Adapters.SQL do
   end
 
   defp query(repo, sql, params, outer_queue_time, mapper, opts) do
-    {pool_mod, pool, timeout} = repo.__pool__
+    {pool_mod, pool, timeout, _} = repo.__pool__
```

Comment:
This was failing on the match, because `repo.__pool__` now returns a tuple with 4 elements, not 3. Do we need to care about this value? It looked like it was an integer value, but I couldn't tell what it meant.